rework_pysatl_mpest/preprocessing/components_family/criterions/sample_criterions.py

```diff
@@ -0,0 +1,175 @@
+"""Module which contains sample features for a mixture classifier"""
+
+__author__ = "Mark Dubrovchenko"
+__copyright__ = "Copyright (c) 2025 PySATL project"
+__license__ = "SPDX-License-Identifier: MIT"
+
+import numpy as np
+from rework_pysatl_mpest.preprocessing.components_family.criterions.abstract_criterion import (
+    ASampleClassifierCriterion,
+)
+from scipy.stats import iqr, kurtosis, skew, zscore
+
+
+class CKurt(ASampleClassifierCriterion):
+    @property
+    def name(self) -> str:
+        return "Kurtosis Criterion"
+
+    def score(self, X: np.ndarray) -> float:
+        result = kurtosis(X)
+        return kurtosis(X) if not np.isnan(result) else 0
+
+
+class CNegativeValue(ASampleClassifierCriterion):
+    @property
+    def name(self) -> str:
+        return "Negative Value Criterion"
+
+    def score(self, X: np.ndarray) -> float:
+        return float(np.min(X) < 0)
+
+
+class CIqr(ASampleClassifierCriterion):
+    @property
+    def name(self) -> str:
+        return "IQR Criterion"
+
+    def score(self, X: np.ndarray) -> float:
+        return iqr(X) / np.median(X)
+
+
+class CKurtMoors(ASampleClassifierCriterion):
+    @property
+    def name(self) -> str:
+        return "Kurtosis Moors Criterion"
+
+    def score(self, X: np.ndarray) -> float:
+        p1, p2, p3, p4, p5, p6 = np.percentile(X, [12.5, 25, 37.5, 62.5, 75, 87.5])
+        result = ((p6 - p2) + (p5 - p3)) / (p4 - p2)
```

Comment:
Idk but the implementation of Moors' Kurtosis seems incorrect according to the standard definition ($ \frac{(E_7 - E_5) + (E_3 - E_1)}{E_6 - E_2} $) (https://scispace.com/pdf/a-quantile-alternative-for-kurtosis-4fg277s0e1.pdf, page 3)
```suggestion
        p1, p2, p3, p5, p6, p7 = np.percentile(X, [12.5, 25, 37.5, 62.5, 75, 87.5])
        result = ((p7 - p5) + (p3 - p1)) / (p6 - p2)
```